src/framework/components/collision/system.js

```diff
@@ -189,6 +191,8 @@ class CollisionSystemImpl {
             enabled: src.data.enabled,
             type: src.data.type,
             halfExtents: [src.data.halfExtents.x, src.data.halfExtents.y, src.data.halfExtents.z],
+            linearOffset: [src.data.linearOffset.x, src.data.linearOffset.y, src.data.linearOffset.z],
+            angularOffset: [src.data.angularOffset.x, src.data.angularOffset.y, src.data.angularOffset.z],
```

Comment:
should there be .w as well, or are these just 3 angles here

Reply:
Nope, that should be 4 as it's a quat